+YTGhost: 283304489!qq.com
+	SEL-Zhejiang University
 YuChaode: yuchaode!huawei.com
 	Huawei Technologies Co. Ltd
 YuPengZTE: YuPengZTE!users.noreply.github.com, yu.peng36!zte.com.cn
diff --git a/developers_affiliations2.txt b/developers_affiliations2.txt
index 835b751c..6408aedf 100644
--- a/developers_affiliations2.txt
+++ b/developers_affiliations2.txt
@@ -22249,6 +22249,10 @@ crellison: crellison!users.noreply.github.com
 	Squarespace Inc. from 2016-12-01 until 2017-08-01
 	Independent from 2017-08-01 until 2018-05-01
 	Squarespace Inc. from 2018-05-01
+crenshaw-dev: crenshaw-dev!users.noreply.github.com
+    Commonwealth Computer Research Inc. until 2020-09-01
+    Independent from 2020-09-01 until 2021-10-01
+    Intuit Inc. from 2021-10-01
 crevillo: crevillo!gmail.com
 	Tanta until 2016-09-01
 	The Cocktail from 2016-09-01 until 2020-02-01
diff --git a/developers_affiliations3.txt b/developers_affiliations3.txt
index 19566648..1265e4f1 100644
--- a/developers_affiliations3.txt
+++ b/developers_affiliations3.txt
@@ -1712,6 +1712,8 @@ dbuschman7: dbuschman7!users.noreply.github.com
 dbussink: d.bussink!gmail.com
 	GitHub Inc. until 2021-07-01
 	PlanetScale from 2021-07-01
+dbyron-sf: dbyron!salesforce.com, dbyron-sf!users.noreply.github.com
+        Independent until 2021-04-12
+        Salesforce.com Inc. from 2021-04-12
 dc0d: dc0d!users.noreply.github.com, kaveh.shahbazian!gmail.com
 	Faragostar until 2014-08-01
 	VSHN AG from 2014-08-01 until 2015-03-01
diff --git a/developers_affiliations4.txt b/developers_affiliations4.txt
index 191df74c..de4b621a 100644
--- a/developers_affiliations4.txt
+++ b/developers_affiliations4.txt
@@ -11932,6 +11932,8 @@ kristrev: kristian.evensen!gmail.com
 	Celerway
 kriswuollett: klw!google.com, kristopher!wuollett.net, kriswuollett!users.noreply.github.com
 	Google LLC
+krisztianfekete: krisztian.fekete!solo.io
+        Solo.io
 krisztian-trl: krisztian-trl!users.noreply.github.com
 	TrivaaLogic
 krkumar: krkumar2!in.ibm.com
@@ -12182,6 +12184,8 @@ kskewes: karl.skewes!gmail.com, kskewes!users.noreply.github.com
 	Rentex until 2018-01-01
 	Uneeq from 2018-01-01 until 2021-08-06
 	Salesforce.com inc. from 2021-08-06
+kskewes-sf: kskewes!salesforce.com, kskewes-sf!users.noreply.github.com
+	Independent until from 2021-08-06
+	Salesforce.com Inc. from 2021-08-06
 kskumgk63: kskumgk63!users.noreply.github.com
 	Independent until 2019-04-01
 	Rakuten Group Inc. from 2019-04-01 until 2019-09-01
diff --git a/developers_affiliations5.txt b/developers_affiliations5.txt
index a40e870b..fb75e3dc 100644
--- a/developers_affiliations5.txt
+++ b/developers_affiliations5.txt
@@ -10219,6 +10219,8 @@ paulmon: paulmon!microsoft.com
 paulnivin: paulnivin!users.noreply.github.com, pfisher!lyft.com
 	Twitter Inc. until 2015-09-01
 	Lyft Inc. from 2015-09-01
+paulofrazao: paulo!weave.works, paulofrazao!users.noreply.github.com, pfrazao!gmail.com
+	Weaveworks Inc.
 paulopontesm: paulo.pontes.m!gmail.com, paulopontesm!users.noreply.github.com
 	Independent until 2014-09-01
 	Hewlett from 2014-09-01 until 2016-06-01
diff --git a/developers_affiliations6.txt b/developers_affiliations6.txt
index 531d1c7e..ec2879fe 100644
--- a/developers_affiliations6.txt
+++ b/developers_affiliations6.txt
@@ -1736,6 +1736,8 @@ shreesha21: shreesha!rancher.com, shreesha21!users.noreply.github.com
 	Hewlett from 2015-03-01 until 2019-02-01
 	Rancher Labs Inc. from 2019-02-01 until 2019-10-01
 	VMware Inc. from 2019-10-01
+shrenikjain38: shrenikj!cisco.com, shrenikjain38!users.noreply.github.com
+	Cisco
 shreyas-s-rao: shreyas-s-rao!users.noreply.github.com, shreyas.sriganesh.rao!sap.com
 	SAP
 shreyasbp: shreyas!linux.vnet.ibm.com, shreyasbp!gmail.com
